owasp-top10-2017-apps/a5/tictactoe/README.md

```diff
@@ -215,7 +218,7 @@ Using Burp Repeater we changed the _user_ parameter by `fry123` and, as shown be
 
   
 
-Now, when the user `fry123` tries to see his statistics by web interface, he will see the modified data:
+Now, when the user `fry123` tries to see his/her statistics by web interface, he/she will see the modified data:
```

Comment:
Typo: `the web interface`.

Actually, I would use `using the web interface` instead of `by the web interface`.

Reply:
Done!